app-admin/testdisk/testdisk-7.1~git.recipe

```diff
@@ -25,62 +25,73 @@ TestDisk can be used to collect detailed information about a \
 non-booting drive which can then be sent to a tech for further analysis. \
 Those more familiar with such procedures should find TestDisk a handy \
 tool in performing onsite recovery."
-HOMEPAGE="http://www.cgsecurity.org/wiki/TestDisk"
-COPYRIGHT="1998-2008 Christophe GRENIER"
+HOMEPAGE="https://www.cgsecurity.org/wiki/TestDisk"
```

Comment:
For the record, the tiny change in 71d61a58ce32eb0f19963fa1ecc87e480bbf1be1 was already in this PR.
I've already fixed the conflict on my local branch, but I'll finish #2614 before updating this one.